open-craft/dalite-ng
# WIP Swift support for Dalite

Testing instuctions: 

1. Install Dalite on your local system. 
2. Configure Swift for dalite (I deliberately didn't add any instructions here, please check instructions from `Readme.md` if something is not obvious please point it and I'll update that file!)
3. You may store static files using Swift, but that's not strictly neccessary . 
4. Create buch of questions, and add images to them. Verify that the images appear in the Swift Container, and are visible when you preview them on Dalite. 
5. You might also test whether it works when you integrate Dalite with LMS, but I'm not sure if it makes sense (that is: nothing can break here...).
